src/components/main/main.jsx

```diff
@@ -148,7 +132,7 @@ const Main = (props) => {
 };
 
 Main.propTypes = {
-  names: PropTypes.arrayOf(PropTypes.string).isRequired,
+  films: PropTypes.array.isRequired,
```

Comment:
```suggestion
  films: PropTypes.arrayOf(filmsShape).isRequired,
```

и где-то в общем месте:
```jsx
import PropTypes from "prop-types";

const filmsShape = PropTypes.shape({
  id: PropTypes.number.isRequired,
  title: PropTypes.string.isRequired,
  ...
}).isRequired;

export {filmsShape};
```